.circleci/config.yml

```
@@ -32,7 +32,7 @@ jobs:
           name: Install napari-dev
           command: |
             . venv/bin/activate
-            python -m pip install -e "napari/[pyside,dev]"
+            python -m pip install -e "napari/[all]"
```

```suggestion
            python -m pip install -e "napari/[pyside,dev]"
```
This is unrelated to this PR, so let's not change the CircleCI workflow which is currently working fine.